.github/workflows/test-elixir.yaml

```diff
@@ -55,8 +55,7 @@ jobs:
           cache-name: cache-elixir-deps
         with:
           path: prql-elixir/deps
-          key:
-            ${{ runner.os }}-mix-${{ env.cache-name }}-${{
+          key: ${{ runner.os }}-mix-${{ env.cache-name }}-${{
```

Comment:
Because pre-commit can't fix files in the `workflows` path, this change in formatting is causing it to fail.

Does `pip install pre-commit` or `brew install pre-commit` not work for you?

(I fixed it myself this time, I'm fine to keep doing it! I ask so you can be independent :) )

Reply:
Oh i forgot the command to do this, thanks max